src/python/grpcio_tests/setup.py

```diff
@@ -37,13 +37,19 @@
 }
 
 INSTALL_REQUIRES = (
-    'coverage>=4.0', 'enum34>=1.0.4',
+    'coverage>=4.0',
+    'enum34>=1.0.4',
     'grpcio>={version}'.format(version=grpc_version.VERSION),
-    'grpcio-channelz>={version}'.format(version=grpc_version.VERSION),
+    # TODO(https://github.com/pypa/warehouse/issues/5196)
+    # Re-enable it once we got the name back
+    # 'grpcio-channelz>={version}'.format(version=grpc_version.VERSION),
     'grpcio-status>={version}'.format(version=grpc_version.VERSION),
     'grpcio-tools>={version}'.format(version=grpc_version.VERSION),
     'grpcio-health-checking>={version}'.format(version=grpc_version.VERSION),
-    'oauth2client>=1.4.7', 'protobuf>=3.6.0', 'six>=1.10', 'google-auth>=1.0.0',
+    'oauth2client>=1.4.7',
```

Comment:
Nit. And honestly barely related to this PR, but it brought it to my attention. Could we do something like

```python
INSTALL_REQUIRES = map(
  lambda x: x.format(version=grpc_version.VERSION),
  (
    'grpcio>={version}',
    ...
  )
)
```
A little bit less boilerplatey.

Reply:
It is a trade off between simplicity and readability. I'm fine with current way of string formatting, and maybe create another PR to change all `setup.py` at once is cleaner.